app/controllers/devise_token_auth/registrations_controller.rb

```diff
@@ -88,22 +80,33 @@ def destroy
       if @resource
         @resource.destroy
         yield @resource if block_given?
-
         render_destroy_success
       else
         render_destroy_error
       end
     end
 
     def sign_up_params
-      params.permit([*params_for_resource(:sign_up), :confirm_success_url])
+      params.permit(*params_for_resource(:sign_up))
     end
 
     def account_update_params
       params.permit(*params_for_resource(:account_update))
     end
 
-    protected
+  protected
```

Comment:
align with `def`

Reply:
this is imho a good way to improve scanning a file.
rubocop would allow you to indent in such a way, Rails favors another but very similar one (but i won't tab twice lol), is this debatable ?
otherwise i will do
(rails exemple : https://github.com/rails/rails/blob/d0d7ce001bdbb0e5551da881caf8156c06238b84/actioncable/lib/action_cable/remote_connections.rb#L56)

Reply:
your way could be better, but we should stick to the project standard, so maybe you can change all the protected/private alignments in another PR